README.md

```diff
@@ -12,6 +12,8 @@ Scanner is an app that demonstrates the event handling capabilities of the Realm
 
 ### Steps To Run
 
+For a fully in-depth tutorial on running the Scanner app along with the Realm Object Server, please see [the Scanner tutorial](https://realm.io/docs/realm-mobile-platform/textscanner/) on the Realm website.
```

Comment:
This link should be changed to https://realm.io/docs/realm-mobile-platform/scanner/ , https://realm.io/docs/realm-mobile-platform/realmscanner/ or https://realm.io/docs/realm-mobile-platform/realm-scanner/